### ISSUE TITLE
Surface Component Font import wrong weight

### ISSUE THREAD

**Issue by j9981** [divriots/story-to-design#196]
## Surface Component Font import wrong weight ##
Thank you for fixing the previous issue. The current surface container can load the Decimal font, and the spacing issue is fixed.

However, it's imported incorrect heading font-weight. It's imported "Thin" instead of "Light" (which is the correct weight in our storybook).

Thank you for your help.

Related ticket: https://github.com/divriots/story-to-design/issues/193

---
Bug report identity: Please do not edit

Task path: hpPpNB3AXDmRHPrweEFJ/xPG4gSrdkxavICSJD1YU/1ndltxymuhq2b

**Comment by muryoh** [comment]
Hi @j9981 and sorry for the delayed replied!

I am currently not able to reproduce, it seems to usse `Decimal Light` as expected:
![image](https://github.com/user-attachments/assets/0725378c-f21e-4986-b79d-a7b9b046620e)

However I may not be using exactly the same `Decimal` fonts you are using locally, maybe that could explain it

Could you please tell me what you're seeing in the Figma fonts picker for `Decimal`? This is what I'm seeing:
![image](https://github.com/user-attachments/assets/159c8a1f-62f7-49c9-8872-5c62388c1faa)

Basically 2 different fonts instead of a single `Decimal` one with 2 different styles

**Comment by j9981** [comment]
Hi @muryoh , thank you for getting back to me.

Here is the screenshot of what I see in my Figma. We are currently only using two weights in our tokens and storybook for Decimal: Light and Book. All the headings should be using Light.
![Screenshot 2024-09-30 at 1 53 38 PM](https://github.com/user-attachments/assets/adde07db-3086-40e6-97f2-d2d00023f599)

Our storybook is using Light Weight Decimal. I believe we didn't have anything set in Thin on Storybook.
![Screenshot 2024-09-30 at 1 59 25 PM](https://github.com/user-attachments/assets/cfd13e1e-c46b-4a15-880f-873f4d3b103f)

**Comment by muryoh** [comment]
Hi,

So indeed we do not have exactly the same fonts locally, that's why I cannot reproduce
I will look into fixing it :+1:
